controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_types.go

```diff
@@ -108,6 +108,8 @@ type KubeadmControlPlaneStatus struct {
 // +kubebuilder:storageversion
 // +kubebuilder:subresource:status
 // +kubebuilder:subresource:scale:specpath=.spec.replicas,statuspath=.status.replicas,selectorpath=.status.selector
+// +kubebuilder:printcolumn:name="Ready",type=boolean,JSONPath=".status.ready",description="Total number of clusters ready"
+//+kubebuilder:printcolumn:name="Initialized",type=boolean,JSONPath=".status.initialized",description="Cluster is in initialized state or not(True/False)"
 
```

Comment:
Would it be worth also adding printcolumn definitions for Replicas, ReadyReplicas, and UpdatedReplicas?

Reply:
@hpandeycodeit  Also UnavailableReplicas.

Reply:
So only UnavailableReplicas in addition to the above change ? or Replicas, ReadyReplicas, and UpdatedReplicas as well?

Reply:
All of them I suppose. So the user can track the states of all the replicas.